app/services/hulu_mapping_service.rb

```diff
@@ -0,0 +1,75 @@
+# Service to sync Hulu API with Kitsu
+class HuluMappingService
+  HULU_URL = 'https://distribution.hulu.com/api/v1/'.freeze
+
+  attr_reader :since
+
+  # Initialize service with optional datetime
+  def initialize(since = nil)
+    @since = since
+  end
+
+  # Grabs all available series from hulu, maps them as wel as their episodes
+  def sync_series_and_episodes
+    ActiveRecord::Base.logger = Logger.new(nil)
+    Chewy.strategy(:bypass)
```

Comment:
Are these two lines intentional or are they something from development that leaked in?